Cargo.toml

```diff
@@ -22,6 +22,9 @@ build = "build.rs"
 all-features = true
 
 [dependencies]
+parity-scale-codec = {package = "parity-scale-codec", optional = true, version = "3.1.5", features = ["max-encoded-len"], default-features = false}
```

Comment:
Nit: could you make these alphabetical in the list? Also - we're making the attributes alphabetical too.
Btw, you shouldn't need to specify the `package` attribute since it is the same name as the crate.

Reply:
Resolved